python3/plugins/extauth-hook-AD.py

```diff
@@ -85,22 +84,14 @@ def __init__(self, path, session, args, ad_enabled=True, load_existing=True, fil
         self._session = session
         self._args = args
         self._lines = []
-        self._backend = self._get_ad_backend()
+        self._backend = ADBackend.BD_WINBIND
```

Comment:
_backend is now only winbind, 
so we do not need `ADBackend` anymore.

Reply:
Thanks, removed.